includes/internal/emails/class-email-customization.php

```diff
@@ -104,34 +105,37 @@ class Email_Customization {
 	 *
 	 * Prevents other instances from being created outside of `self::instance()`.
 	 *
-	 * @param \Sensei_Settings $settings Sensei_Settings instance.
+	 * @param Sensei_Settings $settings Sensei_Settings instance.
+	 * @param Sensei_Assets   $assets Sensei_Assets instance.
 	 */
-	private function __construct( \Sensei_Settings $settings ) {
+	private function __construct( Sensei_Settings $settings, Sensei_Assets $assets ) {
 		$repository                 = new Email_Repository();
 		$this->post_type            = new Email_Post_Type();
 		$this->settings_menu        = new Settings_Menu();
 		$this->settings_tab         = new Email_Settings_Tab( $settings );
 		$this->blocks               = new Email_Blocks();
-		$this->email_sender         = new Email_Sender( $repository, $settings );
+		$this->email_sender         = new Email_Sender( $repository, $settings, $assets );
```

Comment:
I think is very weird to add assets to an e-mail sender, should we move it to another class?  Sensei_Editor ?

Reply:
Yeah, agree, it is weird. I did a workaround in https://github.com/Automattic/sensei/pull/6583/commits/4db39ae72ea1b3446844dfcafbda12574fc8f990.